site/components/src/main/java/uk/nhs/digital/ps/beans/HippoBeanHelper.java

```diff
@@ -27,6 +27,20 @@ public static boolean isRootFolder(HippoBean folder) {
         return folder.isSelf(siteContentBaseBean);
     }
 
+    public static HashMap<String, String> getFullTaxonomyMap() {
+
+        // Lookup Taxonomy Tree
+        TaxonomyManager taxonomyManager = HstServices.getComponentManager().getComponent(TaxonomyManager.class.getName());
+        Taxonomy taxonomyTree = taxonomyManager.getTaxonomies().getTaxonomy(PUBLICATION_TAXONOMY);
+
+        HashMap<String, String> taxMap = new HashMap<>();
```

Comment:
Suggestion:  we could use Stream APIs here

Reply:
Refactored the method out, so no longer need this